src/pages/Subscription/Subscription.scss

```diff
@@ -0,0 +1,49 @@
+.info {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 0 auto;
+
+  h1 {
+    font-size: 30px;
+    margin-bottom: 10px;
+
+    &::after {
+      content: ".";
+      color: blue;
+    }
+  }
+
+  p {
+    font-size: 40px;
+    margin: 10px 0px !important;
+    color: #919191;
+
+    &::after {
+      content: "soon!";
+      font-weight: bold;
+      color: black;
+    }
+  }

```

Comment:
In that case, I think it's better to put it directly on HTML than style it using a CSS class.
If someone else is looking at your page, it's hard to figure out where this `soon` is coming from.

What you think?

Reply:
so, is best to put back the "soon" on the html, and remove this content?